Makefile

```diff
@@ -48,7 +48,7 @@ check:
 ## check-go: download and run the go linter.
 .PHONY: check-go
 check-go: ## - Run golangci-lint
-	@curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.44.2
+	@curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.55.2
```

Comment:
I don't know if anyone is actually using this target. Let me double check that, it might make sense to delete the target instead. It can't get out of sync if it doesn't exist anymore.

Reply:
I used it myself so personally think it's useful to keep in but LMK if you want me to repurpose this PR to delete it.

Reply:
Sure, it doesn't cost us anything to have it here if it makes it easier for external contributors to figure out what to do.